### PR TITLE
GraphQl: Use compiler pass only when enabled

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | na

Compiler passes ran with no graphql which was producing errors.

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/GraphQlMutationResolverPass.php [hunk 1]
```diff
@@ -31,6 +31,10 @@ final class GraphQlMutationResolverPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
+        if (!$container->getParameter('api_platform.graphql.enabled')) {
```
Comment (vincentchalamon) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/GraphQlMutationResolverPass.php]: You should move this check in [ApiPlatformBundle::build](https://github.com/api-platform/core/blob/be932e901402000adf54fa38e0b741f36474be79/src/Bridge/Symfony/Bundle/ApiPlatformBundle.php#L48), so the CompilerPass won't be executed (same with other GraphQl CompilerPasses)

Comment (soyuka) [reply src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/GraphQlMutationResolverPass.php]: thanks @vincentchalamon I updated the PR, I did the same thing for elasticsearch.

**Comment by teohhanhui** [comment]
@soyuka I think you merged the wrong commit. :laughing:

**Comment by soyuka** [comment]
Yes parameters are not ready there.

**Pull request merged**